app/templates/application.hbs

```diff
@@ -1,5 +1,4 @@
-<HeadLayout />
```

Comment:
It looks like deleting `<HeadLayout>` resulted in a breaking change for us. Previously, we could see 3 `<meta>` tags in the deploy preview website:

<img width="640" alt="Screen Shot 2021-01-12 at 6 38 59 PM" src="https://user-images.githubusercontent.com/16869656/104351358-93ee5680-5505-11eb-8eb7-2bee0fb40c25.png">

Now that `ember-page-title` no longer depends on `ember-cli-head`, I think we'll need to find another way to dynamically create these meta tags.

References:

- https://github.com/ember-cli/ember-page-title/issues/201
- https://github.com/emberjs/rfcs/pull/645#issuecomment-698391919

Reply:
I asked for help on Discord: https://discord.com/channels/480462759797063690/480501885837770763/798610097319444520